notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "b6dbb8aa-c070-4e7d-8b7a-64588e7f0da4",
```

Comment:
<p>Why do you need to use opencv to load an image? Is it possible to change it to PIL to remove <code>transform_fn</code>?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>